.landscape.yml

```diff
@@ -0,0 +1,2 @@
+python-targets:
```

Comment:
Sadly it doesn't seem to be detected

Go to the "Errors" page in landscape.io and you will see a lot of syntax errors for all type returns.

landscape.io seems to have bugged out currently though and accessing some pages give me 500 http responses and some tests have been executed for 2 hours now so i'm not sure if this .landscape.yml file even fixes it yet.

> If you are currently writing a project which is Python 3 compatible, then you are probably getting warnings from Landscape about 'syntax errors'. This is because until now Landscape was always checking using Python 2 and therefore 3-only syntax such as yield from was marked as an error.

https://blog.landscape.io/python-3-support.html